src/Engine/Audio/SteamAudio.cpp

```diff
@@ -5,15 +5,35 @@
 #include <MemTrackInclude.hpp>
 #endif
 
+using namespace Audio;
+
 SteamAudio::SteamAudio() {
     playerPos = IPLVector3{ 0,0,0 };
     playerDir = IPLVector3{ 1,0,0 };
     playerUp = IPLVector3{ 0,1,0 };
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)